### ISSUE TITLE
Default .sailsrc contains grunt:false

### ISSUE THREAD

**Issue by dvarnai** [balderdashy/sails#3013]
As you can see in the title, the default .sailsrc after creating a new project contains grunt: false. I've verified this issue on CentOS7, Debian7 and Windows 8.1. Also while looking for a fix to the problem it caused I bumped into other people on IRC who had the same problem on Arch. Although it seems that it doesn't always happen, as there was a guy who tried what we did and his didn't contain it.

steps to reproduce:

```
npm -g install sails
sails new test
cat test/.sailsrc
```

expected output:

```
{
  "generators": {
    "modules": {}
  }
}
```

actual result:

```
{
  "generators": {
    "modules": {}
  },
  "hooks": {
    "grunt": false
  }
}
```

**Comment by tarlepp** [comment]
Yep that other guy was me, below is result with my Ubuntu virtualbox setup.

```bash
wunder@wunder-VirtualBox:~$ sails --version
0.11.0
wunder@wunder-VirtualBox:~$ cd projects/
wunder@wunder-VirtualBox:~/projects$ sails new testapp
info: Created a new Sails app `testapp`!
wunder@wunder-VirtualBox:~/projects$ cd testapp/
wunder@wunder-VirtualBox:~/projects/testapp$ more .sailsrc 
{
  "generators": {
    "modules": {}
  }
}
wunder@wunder-VirtualBox:~/projects/testapp$ node --version
v0.12.2
```

**Comment by dvarnai** [comment]
On a fresh Ubuntu Server 15.04 install that I just did to test this:

http://pasteboard.co/1dQxVAqe.png

**Comment by ivanhuay** [comment]
+1

**Comment by killerhis** [comment]
I can confirm the issue on OS X Yosemite and Windows 8.1 using the latest version of sails... was a real headache to figure this out

**Comment by dennisver** [comment]
+1

**Comment by albertbenitez** [comment]
how can we solve this ??

**Comment by kennysmoothx** [comment]
+1

**Comment by killerhis** [comment]
Solution for this (until they fix it) is pretty easy

Just delete the "hooks" part array with the "grunt": false property, save it and you're ready to go

**Comment by tarlepp** [comment]
yep, this is only "problem" when you're creating new sails app, so not hard to change that file.

**Comment by sgress454** [comment]
Sorry about this folks, prematurely pushed out a version of `sails-generate-new`.  Should be fixed now in version v0.10.21.
